docs/pages/transactions/send-batch-transactions/api.mdx

```diff
@@ -0,0 +1,64 @@
+<Info>
+  The examples provided use [foundry](https://getfoundry.sh/) and
+  [jq](https://jqlang.org/) to prepare and parse
+</Info>
+
+See the [`wallet_prepareCalls` API
+reference](/docs/wallets/api/smart-wallets/wallet-api-endpoints/wallet-api-endpoints/wallet-prepare-calls)
+for full descriptions of the parameters used in the following example.
+
+<Steps>
+  <Step title="(If Needed) Request Account">
+    If the user does not yet have a smart account, you must create one.
+```bash twoslash
+  ACCOUNT_ADDRESS=$(curl --request POST \
+  --url https://api.g.alchemy.com/v2/$ALCHEMY_API_KEY \
+  --header 'accept: application/json' \
+  --data '
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "method": "wallet_requestAccount",
+  "params": [
+    { 
+      "signerAddress": "'$SIGNER_ADDRESS'"
+    }
+  ]
+}
+' | jq -r '.result.accountAddress')
+```
+  </Step>
+  <Step title="Prepare Calls">
+    Prepare calls with multiple calls specified in the batch.
+```bash twoslash
+curl --request POST \
+  --url https://api.g.alchemy.com/v2/$ALCHEMY_API_KEY \
+  --header 'accept: application/json' \
+  --data '
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "method": "wallet_prepareCalls",
+  "params": [
+    {
+      "calls": [
+        {
+          "to": "0xCFf7C6dA719408113DFcb5e36182c6d5aa491443",
+          "data": "'$(cast calldata "approve(address,uint256)" 0xB0AEC4c25E8332256A91bBaf169E3C32dfC3C33C $(cast parse-units 5000 6))'"  
+        },
+        {
+          "to": "0xB0AEC4c25E8332256A91bBaf169E3C32dfC3C33C",
+          "data": "'$(cast calldata "swapUSDCtoWETH(uint256,uint256)" $(cast parse-units 5000 6) $(cast parse-units 1 18))'"
+        } 
+      ],
+      "from": "'$ACCOUNT_ADDRESS'",
+      "chainId": "'$CHAIN_ID'"
+    }
+  ]
+}'
+```
+  </Step>
+  <Step title="Sign & Send">
+    Sign the returned signature request and send using `wallet_sendPreparedCalls`.
```

Comment:
Should we include the steps here?

Reply:
That makes sense, I think it's probably fine to just link to the send transactions section or leave as is for now